lib/backend/k8s/conversion_test.go

```diff
@@ -429,3 +431,38 @@ var _ = Describe("Test Namespace conversion", func() {
 		Expect(outboundRules[0].Action).To(Equal("allow"))
 	})
 })
+
+var _ = Describe("Test Node conversion", func() {
+
+	It("Should parse a k8s Node to a Calico Node", func() {
+		l := map[string]string{"net.beta.kubernetes.io/role": "master"}
+		node := k8sapi.Node{
+			ObjectMeta: k8sapi.ObjectMeta{
+				Name: "TestNode",
+				Labels: l,
+				ResourceVersion: "1234",
+			},
+			Spec: k8sapi.NodeSpec{},
+			Status: k8sapi.NodeStatus{
+				Addresses: []k8sapi.NodeAddress{k8sapi.NodeAddress{
```

Comment:
Can we add a test for:

- when there is an ExternalIP and an InternalIP provided?
- when there is just an ExternalIP provided?
- when there are no IPs provided?